src/t8_cmesh/t8_cmesh_vtk_to_t8/t8_cmesh_vtk_polydata.cxx

```diff
@@ -0,0 +1,113 @@
+/*
+This file is part of t8code.
+t8code is a C library to manage a collection (a forest) of multiple
+connected adaptive space-trees of general element classes in parallel.
+
+Copyright (C) 2015 the developers
+
+t8code is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+t8code is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with t8code; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+#if T8_WITH_VTK
+#include <t8_cmesh_vtk_polydata.hxx>
+#include <vtkPolyData.h>
+#include <vtkBYUReader.h>
+#include <vtkOBJReader.h>
+#include <vtkPLYReader.h>
+#include <vtkPolyDataReader.h>
+#include <vtkSTLReader.h>
+#include <vtkXMLPolyDataReader.h>
+
+int
+t8_read_poly (const char *filename, vtkSmartPointer < vtkPolyData > grid)
```

Comment:
```bash
In file included from ./../t8code/src/t8_cmesh/t8_cmesh_vtk_to_t8/t8_cmesh_vtk_polydata.cxx:25:
./../t8code/src/t8_cmesh/t8_cmesh_vtk_to_t8/t8_cmesh_vtk_polydata.hxx:50:53: error: 'vtkPolyData' was not declared in this scope; did you mean 'vtkPolyData_h'?
   50 |                                   vtkSmartPointer < vtkPolyData > grid);
      |                                                     ^~~~~~~~~~~
      |                                                     vtkPolyData_h
./../t8code/src/t8_cmesh/t8_cmesh_vtk_to_t8/t8_cmesh_vtk_polydata.hxx:50:65: error: template argument 1 is invalid
   50 |                                   vtkSmartPointer < vtkPolyData > grid);
```

Reply:
Which vtk version are you using, and have you linked it properly?